code/modules/antagonists/heretic/items/eldritch_painting.dm

```diff
@@ -0,0 +1,418 @@
+
+// The sister and He Who Wept eldritch painting
+// All eldritch paintings are based on this one, with some light changes
+/obj/item/wallframe/painting/eldritch
+	name = "The sister and He Who Wept"
+	desc = "A beautiful artwork depicting a fair lady and HIM, HE WEEPS, I WILL SEE HIM AGAIN."
+	icon = 'icons/obj/signs.dmi'
+	resistance_flags = FLAMMABLE
+	flags_1 = NONE
+	icon_state = "eldritch_painting_weeping"
+	result_path = /obj/structure/sign/painting/eldritch
+	pixel_shift = 30
+
+/obj/structure/sign/painting/eldritch
+	name = "The sister and He Who Wept"
+	desc = "A beautiful artwork depicting a fair lady and HIM, HE WEEPS, I WILL SEE HIM AGAIN. Destroyable with wirecutters."
+	icon = 'icons/obj/signs.dmi'
+	icon_state = "eldritch_painting_weeping"
+	custom_materials = list(/datum/material/wood =SHEET_MATERIAL_AMOUNT)
+	resistance_flags = FLAMMABLE
+	buildable_sign = FALSE
+	// The list of canvas types accepted by this frame, set to zero here
+	accepted_canvas_types = list()
+	// This stops people hiding their sneaky posters behind signs
+	layer = CORGI_ASS_PIN_LAYER
+	// A basic proximity sensor
+	var/datum/proximity_monitor/painting_proximity_sensor
+	// For changing sensor types on subtypes
+	var/sensor_type = /datum/proximity_monitor/advanced/eldritch_painting
+	// Set to false since we don't want this to persist
+	persistence_id = FALSE
+
+// Mood applied for ripping the painting
+// These moods are here to check hallucinations and provide easier user feedback
+/datum/mood_event/eldritch_painting
+	description = "YOU, I SHOULD NOT HAVE DONE THAT!!!"
+	mood_change = -6
+	timeout = 3 MINUTES
+
+/datum/mood_event/eldritch_painting/weeping
+	description = "HE IS HERE, AND HE WEEPS!"
+	mood_change = -3
+	timeout = 10 SECONDS
+
+/datum/mood_event/eldritch_painting/weeping_heretic
+	description = "Oh such arts! They truly inspire me!"
+	mood_change = 5
+	timeout = 5 MINUTES
+
+/datum/mood_event/eldritch_painting/weeping_withdrawl
+	description = "My mind is clear from his influence."
+	mood_change = 1
+	timeout = 5 MINUTES
+
+/obj/structure/sign/painting/eldritch/Initialize(mapload, dir, building)
+	if(sensor_type)
+		painting_proximity_sensor = new sensor_type(_host = src, range = 7, _ignore_if_not_on_turf = TRUE)
+	return ..()
+
+/obj/structure/sign/painting/eldritch/wirecutter_act(mob/living/user, obj/item/I)
+	user.add_mood_event("ripped_eldritch_painting", /datum/mood_event/eldritch_painting)
+	to_chat(user, span_notice("laughter echoes through your mind"))
```

Comment:
Is this supposed to be a balloon alert or is it just a weirdly formatted message
```suggestion
	to_chat(user, span_notice("Laughter echoes through your mind..."))
```